content/references/protocol-reference/ledger-data/ledger-object-types/accountroot.ja.md

```diff
@@ -56,6 +56,22 @@ labels:
 | `WalletLocator`               | 文字列    | Hash256           | _（省略可）_ **廃止予定**。使用しないでください。 |
 | `WalletSize`                  | 数値    | UInt32            | _（省略可）_ **廃止予定**。使用しないでください。 |
 
+## Special AMM AccountRoot Objects
+<!-- TODO: translate this section -->
+
+{% include '_snippets/amm-disclaimer.md' %}
+
+Automated Market Makers (AMMs) use an AccountRoot object to issue their LP Tokens and hold the assets in the AMM pool, in addition to the [AMM object][] for tracking some of the details of the AMM. The address of an AMM's associated AccountRoot is randomized so that users cannot identify and fund the address in advance of the AMM being created. Unlike normal accounts, AMM AccountRoots are created with the following settings:
```

Comment:
Do we need to make mention of the AMM object here? It has its own page going over that info.

Reply:
I think it's useful to mention it here in case people read this one first—just so people know that an AMM is always two ledger entries, one of each type.